spec/features/process_a_request_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'Processing a request', js: true do
+  include ActiveJobHelper
+
+  let(:vst) { create(:visit) }
```

Comment:
Yeah, it's unfortunate. I also considered `vizit` (à la `klass`) but thought it would be even worse!